api/src/group_create.ts

```diff
@@ -46,63 +46,62 @@ function validateRequestBody(body: any): Result.Type<RequestBody> {
 function mkSwaggerSchema(server: FastifyInstance) {
   return {
     beforeHandler: [(server as any).authenticate],
-    description: "Create a new group.",
-    tags: ["global"],
-    summary: "Create a new group",
-    security: [
-      {
-        bearerToken: [],
-      },
-    ],
-    body: {
-      type: "object",
-      required: ["apiVersion", "data"],
-      properties: {
-        apiVersion: { type: "string", example: "1.0" },
-        data: {
-          type: "object",
-          additionalProperties: false,
-          required: ["group"],
-          properties: {
-            group: {
-              type: "object",
-              required: ["id", "displayName", "users"],
-              properties: {
-                additionalProperties: false,
-                id: { type: "string", example: "Manager" },
-                displayName: { type: "string", example: "All Manager Group" },
-                users: { type: "array", items: { type: "string" } },
-              },
-            },
-          },
+    schema: {
+      description: "Create a new group.",
+      tags: ["global"],
+      summary: "Create a new group",
+      security: [
+        {
+          bearerToken: [],
         },
-      },
-    },
-    response: {
-      200: {
-        description: "successful response",
+      ],
+      body: {
         type: "object",
+        required: ["apiVersion", "data"],
         properties: {
           apiVersion: { type: "string", example: "1.0" },
           data: {
             type: "object",
+            additionalProperties: false,
+            required: ["group"],
             properties: {
-              created: { type: "boolean", example: "true" },
+              group: {
+                type: "object",
+                required: ["id", "displayName", "users"],
+                properties: {
+                  additionalProperties: false,
+                  id: { type: "string", example: "Manager" },
+                  displayName: { type: "string", example: "All Manager Group" },
+                  users: { type: "array", items: { type: "string" } },
+                },
+              },
             },
           },
         },
       },
-      401: NotAuthenticated.schema,
-      409: {
-        description: "Group already exists",
-        type: "object",
-        properties: {
-          apiVersion: { type: "string", example: "1.0" },
-          error: {
-            type: "object",
-            properties: {
-              code: { type: "string", example: "409" },
-              message: { type: "string", example: "User already exists." },
+      response: {
+        200: {
+          description: "successful response",
+          type: "object",
+          properties: {
+            apiVersion: { type: "string", example: "1.0" },
+            data: {
+              type: "object",
```

Comment:
We should return information about the created group on success

Reply:
what kind of information should be returned? 
for e.g. projects we return the project id - in the case of groups, the group id is part of the request. should it then still be returned?

Reply:
changed returned information to following object:
```
group: {
                  type: "object",
                  properties: {
                    id: { type: "string", example: "d0e8c69eg298c87e3899119e025eff1f" },
                    displayName: { type: "string", example: "mstein" },
                    users: { type: "array", items: { type: "string" } },
                  },
                },
```